spec/requests/api/articles_request_spec.rb

```diff
@@ -17,9 +17,44 @@
     end
 
     describe 'response has keys' do
-      it 'has key :title' do
+      it ':title' do
         expect(response_json['articles'][0]).to have_key 'title'
       end
     end
   end
+
+  describe 'GET /api/articles/:id' do
```

Comment:
Create a article folder within the request folder. Have a index_spec and a show_spec. 

We dont want to test these two different action in the same spec file.

Reply:
Will refactor

Reply:
Refactor done